src/dma.rs

```diff
@@ -738,7 +742,7 @@ macro_rules! on_channel {
     };
 }
 
-#[cfg(feature = "stm32f303")]
+#[cfg(any(feature = "stm32f303", feature = "stm32f302"))]
```

Comment:
Actually I think this whole `#[cfg]` isn't needed at all. Since we already have the gate in lib.rs.